data/json/items/tool/electronics.json

```diff
@@ -914,5 +914,109 @@
       "menu_text": "Turn off",
       "type": "transform"
     }
+  },
+  {
+    "id": "20000mah_powerbank_off",
+    "type": "TOOL",
+    "name": { "str": "cheap powerbank (UPS) (off)", "str_pl": "cheap powerbanks (UPS) (off)" },
+    "description": "A cheap rechargable powerbank that was quite common before the cataclysm. Who would have thought it'd take an apocalypse to reduce screen time? You could use it to charge your other UPS devices.",
+    "weight": "412 g",
+    "volume": "220 ml",
+	"longest_side": "13 cm",
```

Comment:
You marked all these bot messages as resolved. I assume you astyled locally and haven't pushed yet?